scm-ui/ui-components/src/repos/diff/DiffFileTree.tsx

```diff
@@ -79,6 +82,7 @@ const TreeNode: FC<NodeProps> = ({ node, parentPath, currentFile, setCurrentFile
         </ul>
       ) : (
         <TreeFile
+          changeType={node.changeType}
```

Comment:
```suggestion
          changeType={node.changeType.toLowerCase() as ChangeType}
```

Since I worked with lowercase below. But I don't think it's very relevant whether this is upper or lower. I just noticed that my suggestion below no longer works otherwise.